src/controller/students/createStudentController.js

```diff
@@ -0,0 +1,17 @@
+import Student from '../../models/student';
+const mongoose = require('mongoose');
+
+export const createStudentController = async (req, res) => {
+    try {
+        const { body } = req;
+        const credentials = process.env.MONGODB_URL;
+        await mongoose.connect(credentials);
```

Comment:
I think there's no need to connect to de database every time you want to create a new student